### PR TITLE
Big Mining Update

### PR DESCRIPTION
**Pull request by Geevies**
* Added the Drill Technician alt-title to Shaft Miner. They come equipped with a kit to set up a barebones drill mining operation.
* Shaft Miners now come with a KA kit that allows them to give slight upgrades to their roundstart Kinetic Accelerator.
* Pumping a kinetic accelerator no longer spams your log, only giving a message every ten seconds since the last one. Sound and charge amount remains the same.
* Fixed being unable to rotate mining braces. Fixed the message saying that you can rotate drill heads.
* Large mining drills now have lights attached to them, which differ in colour and brightness depending on current activity.
* Added the Orbital Drill Dropper to the mining vendor, for 1250 mining points a pop.
* Mining light beacons now come in stacks of 25 instead of stacks of 5. This'll make them slightly more useful for showing a taken path.
* Ore satchels no longer show that they have a warp pack installed on examine, if they do not have one installed.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/mining/machine_vending.dm [hunk 1]
```diff
@@ -5,57 +5,58 @@
...
 	new /datum/data/mining_equipment("Drone KA Upgrade",			/obj/item/device/mine_bot_upgrade/ka,						10,					800),
-	new /datum/data/mining_equipment("Ore Summoner",				/obj/item/oreportal,									35,					800),
-	new /datum/data/mining_equipment("Lazarus Injector",			/obj/item/lazarus_injector,							25,					1000),
+	new /datum/data/mining_equipment("Ore Summoner",				/obj/item/oreportal,										35,					800),
+	new /datum/data/mining_equipment("Lazarus Injector",			/obj/item/lazarus_injector,									25,					1000),
 	new /datum/data/mining_equipment("Industrial Drill Head",		/obj/machinery/mining/drill,								-1,					1000,	1),
-	new /datum/data/mining_equipment("Super Resonator",				/obj/item/resonator/upgraded,						10,					1250),
-	new /datum/data/mining_equipment("Diamond Pickaxe",				/obj/item/pickaxe/diamond,							10,					1500),
-	new /datum/data/mining_equipment("Thermal Drill",				/obj/item/gun/energy/vaurca/thermaldrill,			5,					3750)
+	new /datum/data/mining_equipment("Orbital Drill Dropper",		/obj/item/device/orbital_dropper/drill,						10,					1250),
```
Comment (Arrow768) [inline code/modules/mining/machine_vending.dm:56]: Bump up the price of that to 3000+.
Drillhead + 2 braces is 2000 (which would be more than the price of a complete drill with orbital drop)

File: html/changelogs/geeves-drill_technician.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+delete-after: True
+
+changes: 
+  - rscadd: "Added the Drill Technician alt-title to Shaft Miner. They come equipped with a kit to set up a barebones drill mining operation."
+  - rscadd: "Shaft Miners now come with a KA kit that allows them to give slight upgrades to their roundstart Kinetic Accelerator."
+  - tweak: "Pumping a kinetic accelerator no longer spams your log, only giving a message every ten seconds since the last one. Sound and charge amount remains the same."
+  - bugfix: "Fixed being unable to rotate mining braces. Fixed the message saying that you can rotate drill heads."
+  - rscadd: "Large mining drills now have lights attached to them, which differ in colour and brightness depending on current activity."
+  - rscadd: "Added the Orbital Drill Dropper to the mining vendor, for 1250 mining points a pop."
```
Comment (Geevies) [inline html/changelogs/geeves-drill_technician.yml:11]:
```suggestion
  - rscadd: "Added the Orbital Drill Dropper to the mining vendor, for 3250 mining points a pop."
```

**Comment by Geevies** [comment]
A custom toolbox sprite added, credits to @fernerr

**Pull request merged**